recipes/pneumo-typer/meta.yaml

```diff
@@ -40,8 +40,8 @@ test:
 
 about:
   home: 'https://www.microbialgenomic.cn/Pneumo-Typer.html'
-  summary: 'Pneumo-Typer is a fast and simple-to-use tool for predicting serotype and determining sequence type (St/cgSt) for Streptococcus pneumoniae.'
-  description: 'Pneumo-Typer is a stand-alone perl application, which requires blat, prodigal, NCBI BLASt+, boat and several perl Modules (GD, GD::SVG) to be installed before use.'
+  summary: 'Pneumo-Typer is a comprehensive prediction and visualization of serotype and sequence type for streptococcus pneumoniae using assembled genomes.'
+  description: 'Pneumo-Typer is a stand-alone perl application, which requires blat, prodigal, NCBI BLAST+, and several perl Modules (GD, GD::SVG) to be installed before use.'
   doc_url: How to install and use Pneumo-Typer at 'https://www.microbialgenomic.cn/Pneumo-Typer.html' and or https://github.com/Xiangyang1984/Pneumo-Typer.
```

Comment:
should be a URL like  `https://www.microbialgenomic.cn/Pneumo-Typer.html`

Reply:
Thanks, we corrected the meta.yaml and rebuilt the software.